WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -67,10 +69,13 @@ class HelpActivity : AppCompatActivity() {
                 emailSuggestion = supportHelper
                         .getSupportEmailAndNameSuggestion(accountStore.account, selectedSiteFromExtras).first
             }
+
             supportHelper.showSupportIdentityInputDialog(this, emailSuggestion, isNameInputHidden = true) { email, _ ->
                 zendeskHelper.setSupportEmail(email)
                 refreshContactEmailText()
+                AnalyticsTracker.track(Stat.SUPPORT_IDENTITY_SET)
             }
+            AnalyticsTracker.track(Stat.SUPPORT_IDENTITY_FORM_VIEWED)
```

Comment:
There are 2 different ways a support identity dialog is shown. One is this one, where the user updates their email, the other is where they set the identity for the first time. This might be a bit nitpicky, but for me, the naming suggests that we are tracking both of these events.

I am just bringing it up to avoid a possible confusion in the future. As long as we are aware of it, I am fine with leaving as is.

Reply:
Good point.  We can iterate and tweak for clarity.